src/app/utils/utils.test.js

```diff
@@ -0,0 +1,19 @@
+import { pluralize } from "./utils";
+
+describe("Util", () => {
+  it("pluralize should correctly pluralize a single item", () => {
```

Comment:
Can you group like tests with another describe....so in this case `describe("pluralize", () =>`